### PR TITLE
[12.0][ADD] web_widget_google_marker_icon_picker

### PR DESCRIPTION
**Pull request by murtuzasaleh**
Google Marker Icon Picker
====================

New widget `google_marker_picker` allowing user to set marker's color from
field Loaded with 20 different marker's color.

### CODE REVIEW

**Comment by murtuzasaleh** [comment]
@dreispt @pedrobaeza @wolfhall @max3903

**Comment by murtuzasaleh** [comment]
@dreispt Can you please review this PR.

File: web_widget_google_marker_icon_picker/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2019, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Google Marker Icon Picker',
```
Comment (dreispt) [inline web_widget_google_marker_icon_picker/__manifest__.py:4]:
```suggestion
    'name': 'Google Marker Icon Picker',
    'summary': 'Google map widget allowing to set marker's color',
```

**Review by dreispt — DISMISSED** [review]

**Review by dreispt — COMMENTED** [review]
LGTM, please fix red build.

**Comment by murtuzasaleh** [comment]
Depend on #223

**Review by dreispt — APPROVED** [review]

**Review by max3903 — APPROVED** [review]

**Comment by max3903** [comment]
@murtuzasaleh Let us know if you need help to fix the Travis and Runbot errors and warnings.

**Comment by dreispt** [comment]
@max3903 I think ocabot doesn't work because the build is red

**Comment by dreispt** [comment]
@max3903 module build is green, the problem is with the main 12.0 branch

**Pull request merged**